app/pb_kits/playbook/pb_selectable_card/selectable_card.rb

```diff
@@ -8,21 +8,21 @@ class SelectableCard
       partial "pb_selectable_card/selectable_card"
 
       prop :checked, type: Playbook::Props::Boolean,
-                  default: false
+                     default: false
       prop :disabled, type: Playbook::Props::Boolean,
-                  default: false
+                      default: false
       prop :icon, type: Playbook::Props::Boolean,
                   default: false
       prop :multi, type: Playbook::Props::Boolean,
-                  default: true
+                   default: true
       prop :input_id, type: Playbook::Props::String
+
+      prop :additional_input_options, type: Playbook::Props::Hash,
```

Comment:
After talking with UX, to keep consistency across all kits with inputs, let's call this prop simply `input_options` instead. That means the two `input_options` methods you have in two of the kits here would be better renamed to `additional_input_options` (switch out the naming).